.github/workflows/testing.yml

```
@@ -0,0 +1,263 @@
+
+      # - name: Upload coverage
+      #   uses: codecov/codecov-action@5a1091511ad55cbe89839c7260b706298ca349f7 # v5.5.1
+      #   with:
+      #     files: coverage/**/coverage-final.json,coverage/**/lcov.info,coverage/coverage-summary.json
+      #     fail_ci_if_error: true
+
+  integration-tests:
+    runs-on: ubuntu-latest
+    needs: [unit-tests]
+    permissions:
+      contents: read # Required for actions/checkout
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)